src/Render.elm

```diff
@@ -17,18 +17,31 @@ import Model exposing (..)
 import Rdap
 
 -- Rendering happens inside a context
-type alias Context =
-    { today : Date -- TODO: this can probably be removed
-    , identifier : Identifier
-    , modificationDate : Date
-    }
-
-mkCtx : Date -> History -> Date -> Context
-mkCtx today history modDate = Context today history.identifier modDate
+type alias Context = {
+    history : History,
+    wasVersion : Maybe Version,
+    isVersion : Maybe Version,
+    versions : List Version,
+    modificationDate : Date
+}
+
+mkCtx : History -> Date -> Context
+mkCtx h modDate =
+    let splited = dropWhile (\v -> is After v.from modDate) h.versions
```

Comment:
Good point. It turns out that splitted is also incorrect. "Split" is an irregular verb and the simple past is "split". I will be fixing it.